src/orga/minutes/2022-02-23.md

```diff
@@ -0,0 +1,210 @@
+**2022-03-23 link --> https://hackmd.io/CqkOlf0XQMOh23Wnmt9qTQ?edit**
```

Comment:
Do we need this here? I don't see it in the other files.

Reply:
Just downloaded the doc as-is and this was there

Reply:
Removed